modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Flamingos2.scala

```diff
@@ -91,24 +91,38 @@ object Flamingos2 {
   }
 
   private def disperserFromObserveType(observeType: String, d: Disperser): Grism = observeType match {
-    case DARK_OBSERVE_TYPE    => Grism.Dark
-    case _                    => d
+    case DARK_OBSERVE_TYPE => Grism.Dark
+    case _                 => d
   }
 
+  private def exposureTime(config: Config): ExtractFailure \/ Duration =
```

Comment:
Ok, I see. The current seqexec apparently uses `calibration:exposureTime` and that's the reason for this change. But if it is not necessary I can remove it